cloudbuild.yaml

```diff
@@ -0,0 +1,57 @@
+steps:
+- id: 'build'
+  name: 'gcr.io/cloud-builders/docker'
+  args:
+  - build
+  - --rm
+  - --no-cache
+  - --tag=gcr.io/$PROJECT_ID/rcran-build:temp
+  - .
+
+# Pushing the intermediate image can be useful 
+# to debug test failures.
+- id: 'intermediate-push'
+  waitFor: ['build']
+  name: 'gcr.io/cloud-builders/docker'
+  args:
+  - push
+  - gcr.io/$PROJECT_ID/rcran-build:temp
+
+- id: 'test-convert'
+  waitFor: ['intermediate-push'] # otherwise the test failure should stop push.
+  name: 'kaggle/rcran-build'
+  args: ['convert', ' --version']
+
+- id: 'test'
+  waitFor: ['intermediate-push'] # otherwise the test failure should stop push.
+  name: 'kaggle/rcran-build'
+  entrypoint: 'bash'
+  args:
+  - '-c'
+  - |
+    set -e
+    Rscript test_build.R
+    # Verify expected test_build.R output is present.
+    [ -s plot1.png ]
+    echo "ok"
+
+- id: 'retag'
+  waitFor: ['test', 'test-convert']
+  name: 'gcr.io/cloud-builders/docker'
+  args:
+  - tag
+  - gcr.io/$PROJECT_ID/rcran-build:temp
+  - gcr.io/kaggle-images/rcran:$_TAG
+
+images: ['gcr.io/kaggle-images/rcran:$_TAG']
+
+options:
+  machineType: 'N1_HIGHCPU_32' # If change, update Ncpus in package_installs.R
```

Comment:
Do you want to leave a note here that we picked this since it's the largest GCB instance size (in its shared pool)?